src/main/java/by/exadel/internship/dto/userDTO/UserDTO.java

```diff
@@ -1,18 +1,19 @@
-package by.exadel.internship.dto;
+package by.exadel.internship.dto.userDTO;
+
 
-import by.exadel.internship.dto.enums.UserRole;
 import by.exadel.internship.dto.internshipDTO.UserInternshipDTO;
+import by.exadel.internship.entity.user.UserRole;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-
+import lombok.experimental.SuperBuilder;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.UUID;
 
 @Data
-@Builder
+@SuperBuilder
```

Comment:
Just remove any builder. It, not needed in an entity

Reply:
done